BackendApartmentReservation/BackendApartmentReservation/DataContracts/UserRoles.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BackendApartmentReservation.DataContracts
+{
+    public enum UserRoles
+    {
+        Admin,
+        User
+    }
+
+    public static class UserRoleString
```

Comment:
Can you tell more about these roles, more specifically, why we need strings there?

Reply:
Added those to be assigned for new users. It's more of a template for future, if we need those. This one can be removed for now, does not affect the PR

Reply:
Lets remove it for now, and add when/if necessary